kubernetes/scenescape-chart/templates/kubeclient-dep.yaml

```
@@ -130,3 +139,12 @@ metadata:
   name: {{ .Release.Name }}-kubeclient-sa
   namespace: {{ .Release.Namespace }}
 {{ end }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  {{ .Release.Name }}-person-detection-retail-0013-config
+  namespace: {{ .Release.Namespace }}
+data:
+  config.yaml: |
+    {{- tpl (.Files.Get "files/person-detection-retail-0013.json") . | nindent 4 }}
```

Let's add ending line
```suggestion
    {{- tpl (.Files.Get "files/person-detection-retail-0013.json") . | nindent 4 }}
    
```